### DISCUSSION TITLE
How to mask a specific domain?

### DISCUSSION THREAD

**Discussion by mayjack0312** [stylus/stylus#2748] [Q&A] [answered]
I want to add a style to a specific address
```
https://www.google.com/
```
But at the same time, I want to block

```
https://www.google.com/search?*
```
How can I shield addresses with the same prefix and different endings?
I modified it for half a day, but it can only be applied at the same time.

**Comment by iChenLei** [comment, marked as answer]
@mayjack0312
 goto https://github.com/openstyles/stylus

**Reply by mayjack0312** [reply]
> @mayjack0312 goto https://github.com/openstyles/stylus

Ok, I'll go and have a look, thank you
